README.md

```diff
@@ -15,11 +15,31 @@ For application features, build instructions and user guides see the [srsRAN doc
 sudo snap install srsran
 ```
 
-### Run the EPC
+## Config files
+
+By default config files are installed from the official srsran repo, for example [enb.conf.example](https://github.com/srsran/srsRAN/blob/master/srsenb/enb.conf.example).
+When running any of the commands these config files are used. 
+To override config files, provide paths to the custom files in the commands.
+
+custom config files and all the files referred in them (ex. `user_db.csv`) must be located in the user's home 
```

Comment:
Is there a reason for the line jump? And the sentence starts with a lowercase

Reply:
✅